rules/1_2_DADA2_ASVs/scripts/2d_big_data_DADA2_merge_chimera.R

```diff
@@ -47,13 +47,17 @@
   unfiltered_length_table <- tapply(colSums(seqtab), factor(nchar(getSequences(seqtab))), sum)
   unfiltered_length_table
   unfiltered_length_table_df <- data.frame(length=as.numeric(names(unfiltered_length_table)), count=unfiltered_length_table)
+  print(unfiltered_length_table_df)
```

Comment:
Do you need that print statement? If you want to leave it, you should probably also print a title for the table first, otherwise it's just cluttering the output.